### PR TITLE
Add RSS button to footer

### PR DESCRIPTION
**Pull request by Southpaw1496**

### CODE REVIEW

File: src/_sass/themes/light/buttons.scss [hunk 1]
```diff
@@ -41,3 +41,14 @@
...
+.button.is-rss {
+  background-color: darkorange;
+  border-color: darkorange;
+  color: white;
+
+  &:hover, &:active {
+    border-color: rgba(255, 140, 0, 0.966);
+    background-color: rgba(255, 140, 0, 0.966);
+  }
+}
```
Comment (gdude2002) [inline src/_sass/themes/light/buttons.scss:54]: You've removed a newline at the end of this file here

Comment (Southpaw1496) [reply src/_sass/themes/light/buttons.scss:54]: What is the point of having a trailing newline?

Comment (gdude2002) [reply src/_sass/themes/light/buttons.scss:54]: It makes the git diffs cleaner if you maintain it

**Review by gdude2002 — CHANGES_REQUESTED** [review]
This is more or less fine, but this isn't _quite_ how we do this. [Take a look at this file](https://github.com/QuiltMC/quiltmc.org/blob/stable/src/_sass/themes/dark/vars.scss) - you'll find that what we do is create a variable (for both the dark and light theme), and then we use the SASS lighten/darken functions to create the required variant. I'd recommend you do that as well - don't worry about the implicit import, that's intentional.

**Comment by gdude2002** [comment]
Good work, thanks for that

**Review by gdude2002 — APPROVED** [review]

**Pull request merged**